components/Dialogs/tests/unit/MDCAlertControllerCustomizationTests.m

```diff
@@ -96,16 +96,33 @@ - (void)testApplyingScrimColorToPresentationController {
   UIColor *scrimColor = [UIColor.orangeColor colorWithAlphaComponent:0.5];
   MDCDialogPresentationController *presentationController = alert.mdc_dialogPresentationController;
   if (presentationController == nil) {
-    return;  // don't fail the test if mdc_dialogPresentationController is empty
+    return;  // don't fail the test if mdc_dialogPresentationController is nil
   }
 
   // When
   presentationController.scrimColor = scrimColor;
 
   // Then
-  XCTAssertEqualObjects(alert.mdc_dialogPresentationController.scrimColor, scrimColor);
+  XCTAssertEqualObjects(presentationController.scrimColor, scrimColor);
 }
 
+- (void)testApplyingScrimColorToAlert {
+  // Given
+  UIColor *scrimColor = [UIColor.blueColor colorWithAlphaComponent:0.3];
+  MDCDialogPresentationController *presentationController = alert.mdc_dialogPresentationController;
+  if (presentationController == nil) {
+    return;  // don't fail the test if mdc_dialogPresentationController is nil
+  }
+
+  // When
+  alert.scrimColor = scrimColor;
+
+  // Then
+  XCTAssertEqualObjects(presentationController.scrimColor, scrimColor);
```

Comment:
_tip:_ Aren't we testing the same thing as the test above? Should we test `XCTAssertEqualObjects(alert.scrimColor, scrimColor);`?

Reply:
Yeah I didn't see the difference in the "When".

Reply:
can we mark this as resolved?